src/RA_Core.cpp

```diff
@@ -40,9 +40,9 @@
 std::wstring g_sHomeDir;
 std::string g_sROMDirLocation;
 
-HMODULE g_hThisDLLInst = nullptr;
-HINSTANCE g_hRAKeysDLL = nullptr;
-HWND g_RAMainWnd = nullptr;
+ra::InstanceH g_hRAKeysDLL;
+ra::InstanceH g_hThisDLLInst;
+ra::AppWndH g_RAMainWnd;
```

Comment:
The default constructor of `std::unique_ptr`, even with a custom deleter is `constexpr` (function pointers/objects are locked in the same location) so this it'll also get rid of warnings in the long run. All this stuff gets deallocated when w/e app is integrated ends.